pkgs/development/libraries/lcrq/default.nix

```diff
@@ -23,6 +23,6 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://librecast.net/lcrq.html";
     license = [ lib.licenses.gpl2 lib.licenses.gpl3 ];
     maintainers = with lib.maintainers; [ albertchae aynish DMills27 jasonodoom jleightcap ];
-    platforms = lib.platforms.gnu;
+    platforms = lib.platforms.unix;
```

Comment:
Just calling out that the bump to `0.1.2` fixes darwin support, so we can change the platform now.